packages/client/test/test-utils/fake/FakeLogger.ts

```diff
@@ -0,0 +1,64 @@
+import { Logger, LogLevel } from '@streamr/utils'
+
+interface LogEntry {
+    message: string
+    level: LogLevel
+}
+
+const LOG_LINE_PREFIX_LENGTH = 56 // Logger prefixes each line with level, timestamp and context
+
+export class FakeLogger implements Omit<Logger, 'getFinalLogger'> {
+
+    private entries: LogEntry[] = []
+    private format: (pattern: string, args: any[], cb: (message: string) => void) => void
+
+    constructor() {
+        // format escape sequences (%s, %o etc.) using real Logger
+        this.format = (pattern: string, args: any[], cb: (message: string) => void) => {
+            const logger = new Logger(module, undefined, 'trace', {
+                write: (line: string) => {
+                    // eslint-disable-next-line
+                    const withoutColors = line.replace(/\x1b\[[0-9]+m/gi, '')
+                    const withoutPrefix = withoutColors.substring(LOG_LINE_PREFIX_LENGTH)
+                    const withoutLinefeed = withoutPrefix.substring(0, withoutPrefix.length - 1)
+                    cb(withoutLinefeed)
+                }
+            })
```

Comment:
I think we should create a simple unit test for `FakeLogger` to ensure that it strips away this info correctly. To guard against situations where we would change the format of the real `Logger` but forget to account for the change here.

Reply:
Added a unit test